.github/workflows/fast_testing.yaml

```diff
@@ -35,4 +35,23 @@ jobs:
       - name: Clone the module
         uses: actions/checkout@v2
 
-      - run: make test
+      - name: Cache rocks
+        uses: actions/cache@v2
+        id: cache-rocks
+        with:
+          path: .rocks/
+          key: cache-rocks-${{ matrix.tarantool }}-01
+
+      - name: Install requirements
+        run: |
+          tarantoolctl rocks install luatest 0.5.0
+        if: steps.cache-rocks.outputs.cache-hit != 'true'
+
+      - name: Export path
+        run: echo $PWD/.rocks/bin >> $GITHUB_PATH
+
+      - name: Run luatest tests
+        run: luatest -v
+
+      - name: Run tap tests
+        run: make test
```

Comment:
For some reasons, you want to create new tests using `luatest` library. It's ok, but mixing different test libraries is not ok. Let's convert TAP tests to luatest's tests.

Reply:
Let's do this in a separate task. Completion of this feature is urgent for the customer.

Reply:
https://github.com/tarantool/expirationd/issues/61

Reply:
Discussed and decided to postpone it and do it in the scope of another ticket.